src/components/patient/AddressPanel.tsx

```diff
@@ -0,0 +1,32 @@
+import { Paper, Typography } from '@mui/material'
+import { DataGrid, GridColumns } from '@mui/x-data-grid'
+import { FC } from 'react'
+import { useAppConfig } from '../../hooks/useAppConfig'
+import PatientRecord from '../../types/PatientRecord'
+
+const AddressPanel: FC<{ data: PatientRecord }> = ({ data }) => {
+  const { getFieldsByGroup } = useAppConfig()
+  const columns: GridColumns = getFieldsByGroup('address').map(
+    ({ fieldName, fieldLabel }) => {
+      return {
+        field: fieldName,
+        headerName: fieldLabel
```

Comment:
```suggestion
        headerName: fieldLabel,
        flex: 1
```
To avoid empty space in the tables